### DISCUSSION TITLE
Where by file extension

### DISCUSSION THREAD

**Discussion by adzubla** [nushell/nushell#5571] [Q&A] [answered]
How can I do something like this?

    ls | where extension == 'mp4' || extension == 'mkv' | select name size

It is analogous to the command below:

    ls | where type == file | select name size

Any thoughts?

**Comment by hustcer** [comment, marked as answer]
`ls | where name ends-with .md` or `ls | where name =~ ".md$"`. See: https://www.nushell.sh/book/operators.html#order-of-operations

**Comment by kubouch** [comment]
Also `path parse` gives you the extension.
